tests/e2e/e2e_setup_test.go

```diff
@@ -184,7 +203,15 @@ func (s *IntegrationTestSuite) SetupSuite() {
 		}
 	}
 
-	s.containerManager, err = containers.NewManager(!s.skipUpgrade)
+	if str := os.Getenv(upgradeVersionEnv); len(str) > 0 {
+		s.upgradeVersion = str
+
+		if s.upgradeVersion != "" {
+			s.T().Log(fmt.Sprintf("upgrade version set to %s", s.upgradeVersion))
+		}
```

Comment:
It is alsways non-empty because we check `len(str) > 0` above

```suggestion
		s.T().Log(fmt.Sprintf("upgrade version set to %s", s.upgradeVersion))
```